server/data/prisonApiClient.ts

```diff
@@ -50,8 +50,7 @@ export default class PrisonApiClient extends RestClient {
     )) as Promise<PrisonInformation>
   }
 
-  // TODO: No longer used - leave as might use in future check
-  async getLatestHdcStatus(bookingId: string, user: User): Promise<HomeDetentionCurfew> {
+  async getLatestHdcStatus(bookingId: string, user?: User): Promise<HomeDetentionCurfew> {
     return (await this.get(
```

Comment:
This is ok. If not called with a user object, it will fail to get the username and use ANONYMOUS to cache a system token in the TokenStore for hmppsRestClient.